Automattic/wp-calypso
# Plans: Remove references to Happy Chat

Part of efforts to sunset Happy Chat and remove all references: pdm0RZ-BU-p2

## Proposed Changes

* Removes FAQ component from Plans page - [it's confirmed dead](https://github.com/Automattic/wp-calypso/pull/78262#issuecomment-1593144469).

## Testing Instructions

- Do a code search and confirm that this is indeed dead code.
- Although E2E tests should take care of this, spot check a few flows: /start, /plans/<site slug>, /setup/newsletter/plans.
